contracts/contracts/GrantRounds.sol

```diff
@@ -0,0 +1,48 @@
+pragma solidity ^0.8.5;
+
+contract GrantRounds {
+  // Grant Round ID - The hash of the grant round metadata
+  bytes32 public grantRoundsId;
+  // Metadata Pointer
+  string public metaPtr;
+
+  // Owner
+  address owner;
+
+  modifier onlyOwner() {
+    require(msg.sender == owner);
+    _;
+  }
+
+  constructor(
+    address _owner,
+    bytes32 _grantRoundsId,
+    string memory _metaPtr
+  ) {
+    owner = _owner;
+    grantRoundsId = _grantRoundsId;
+    metaPtr = _metaPtr;
+  }
+
+  // State
+  enum State {
```

Comment:
I'm not convinced we need these explicitly stored states. We might be able to get away with two parameters: `startTimestamp` and `endTimestamp`, with the state of the round derived from these values and the current blocktime.